### DISCUSSION TITLE
Does actsion/cache/save guarantee consistency?

### DISCUSSION THREAD

**Discussion by brianjmurrell** [community/community#160596] [Actions]
If I have two jobs running at the same time, racing with each other and both call `actions/cache/save` with the same key at the same moment does _actions/cache_ guarantee that those two processes will not corrupt the cache entry being saved by both?  I.e. one of the `save`rs wins and has their data cached completely and coherently without the other `save`r being able to simultaneously `save` into the same cache slot and corrupting it.

**Comment by Dinara255** [comment]
😵‍💫
